config.js

```diff
@@ -38,4 +38,13 @@ try {
 	//it is okay if no override file is provided
 }
 configInput.colors = getChalkColors(configInput, providedConfig);
+configInput._logLevels = {

```

Comment:
This looks good for me.
Why did you said:
> not sure if they actually make sense

I may not be thinking right. Can you explain?